CHANGELOG.md

```diff
@@ -2,33 +2,37 @@
 
 ## Unreleased
 
-## [v0.13.0] (2023-10-01)
```

Comment:
you dropped the version header, you need to put that back.

Reply:
Oops! Thanks for catching that ;)
It's now reverted.